app/funcs/validator_functions.py

```
@@ -163,6 +163,13 @@ def format_zipcode(column, zipcode: str) -> str:
             }
         )
 
+def check_contains_factory(match_string: str):
+    def check_contains(value: str) -> str:
+        if value and match_string not in value:
+            raise ValueError(f"Value must contain '{match_string}'")
```

Ruff (0.14.7)

Avoid specifying long messages outside the exception class

(TRY003)